### ISSUE TITLE
Potential problems during live upgrade because of using supervisor3

### ISSUE THREAD

**Issue by tothlac** [klarna/supervisor3#21]
The problem is explained in an other ticket created into the repository of brod: https://github.com/klarna/brod/issues/381

**Comment by suexcxine** [comment]
+1 with this, I also encountered this problem,
```
2020-04-03 14:09:48.123 [error]  #PID<0.1607.0> release_handler: cannot find top supervisor for application :brod
```

**Comment by tothlac** [comment]
@suexcxine It's good to know others are also facing this problem. However it is important only if you want to fully support live release upgrades. For us this is the case.

**Comment by mikpe** [comment]
Fixed by 1.1.11 which has been tagged and published to hex.  (1.1.10 also had the fix, but couldn't be published due to a version mismatch.)
